client/src/main/proto/spine/client/query.proto

```diff
@@ -121,8 +121,8 @@ message ResponseFormat {
     google.protobuf.FieldMask field_mask = 1;
 
     // An order for entities returned by the Query.
-    OrderBy order_by = 2 [(validate) = true,
-                          (if_invalid).msg_format = "Invalid query order"];
+    repeated OrderBy order_by = 2 [(validate) = true,
```

Comment:
This is an important extension of our query language. Please don't forget to include it into the release notes.